src/devtools/client/debugger/src/actions/breakpoints/breakpoints.ts

```diff
@@ -200,6 +200,9 @@ export function toggleBreakpointAtLine(cx: Context, line: number): UIThunkAction
 }
 
 export function updateHoveredLineNumber(line: number): UIThunkAction<Promise<void>> {
+  // if you are hovering a line or a print statement panel that has hits, we
+  // will show those hits on the global timeline. In order to do that, we need
+  // to store the editor's hovered line number in global state.
```

Comment:
💬 tbh I hate the we have to do that :(

Reply:
Yeah, I don't *love* it, but everytime I open this file I'm like "Why the heck are we writing a hover event to global state?" and then I remember, so I figured I would save future me some trouble.